scripts/compile-binaries.sh

```diff
@@ -162,6 +162,15 @@ compile_prusa_buddy() {
     cp /home/pi/klipper/out/klipper.bin /home/pi/printer_data/firmware_binaries/firmware-prusa-buddy.bin
```

Comment:
Just cought this one from the earlier PR, change `printer_data` to `printer_data/config`

Reply:
Done